src/converters/degiroConverterV2.test.ts

```diff
@@ -5,13 +5,21 @@ import { GhostfolioExport } from "../models/ghostfolioExport";
 describe("degiroConverterV2", () => {
 
   beforeEach(() => {
-   // jest.spyOn(console, "log").mockImplementation(jest.fn());
+    // jest.spyOn(console, "log").mockImplementation(jest.fn());
+    // Mock YahooFinanceService - we don't want to make real requests and don't care about the result.
+    jest.spyOn(YahooFinanceService.prototype, "getSecurity").mockResolvedValue({
+      symbol: "AAPL",
```

Comment:
And another using the "Start a review" button and then reviewing using the "Review changes" button at the top right of the code tab.